python/lsst/ip/diffim/detectAndMeasure.py

```diff
@@ -22,6 +22,8 @@
 import numpy as np
 
 import lsst.afw.detection as afwDetection
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath  # FOR TESTING BINNING
```

Comment:
Do we still need the clarifying comment since afwMath is necessary for making the binned image?